### ISSUE TITLE
update the views.py to ensure that customer satisfaction section only pushes to staging 

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#689]
The customer satisfaction entries in the [views.py](https://github.com/18F/pulse/blob/adding-custsat-section/app/views.py) should be modified like the a11y section so that that section doesn't push to production.

**Comment by micahsaul** [comment]
This was addressed in #690
